template/example/nuxt.config.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
     resourceHints: false
   },
   modules: [
-    '@@'
+    { handler: require('../') }
```

Comment:
Would `require('@@')` actually work? 🤔

Reply:
@ricardogobbosouza that'd be awesome 🙌 

Because we can keep the same shortcut then and don't have to bother with relative paths